api/grpcserver.go

```diff
@@ -347,3 +349,15 @@ func (svr *GrpcServer) GetLogs(ctx context.Context, in *iotexapi.GetLogsRequest)
 	}
 	return &iotexapi.GetLogsResponse{Logs: ret}, err
 }
+
+// TODO: remove internal call GetActionByActionHash()
+
+// GetActionByActionHash returns action by action hash
+func (svr *GrpcServer) GetActionByActionHash(h hash.Hash256) (action.SealedEnvelope, error) {
```

Comment:
i am thinking move this to `apiServerV2.go`?
so it becomes an internal/utility func of api, not GrpcServer